uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Service/DrupalUserServiceTest.php

```diff
@@ -69,4 +69,22 @@ protected function cleanup() {
     }
   }
 
+  private function createUserAndActivities() {
```

Comment:
I see some problems here:
- Having an "And" in the method name, usually, means a method is doing 2 or more things. Which is exactly what is happening here. Creating the contact doesn't have anything to do with the activity types. Since it's just a single line of code, it can be called directly from the `setUp()` method. 
- The method doesn't really create Activities, but Activity Types
- Why do you need to manually create these Activitiy Types in the first place? Aren't they created by some upgrader in some extension?